src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,62 @@
+#include "whitenoiseeffect.h"
```

Comment:
I think we always use includes relative to our `src/` directory:

```suggestion
#include "effects/builtin/whitenoiseeffect.h"
```